docs/content/faq/_index.md

```diff
@@ -92,6 +92,16 @@ Modules that won't have a module owner associated with them after they have been
 
 The AVM core team plans to have migrated the majority of the CARML modules to AVM by the end of 2023.
 
+### Terraform Timeline and Approach
+
+As the [AVM core team](Azure-Verified-Modules/specs/shared/team-definitions/#avm-core-team) is not directly responsible for the development of the modules (that's the responsibility of the [module owners](/Azure-Verified-Modules/specs/shared/team-definitions/#module-owners)), there's no specific timeline available for when the first Terraform modules will be published.
```

Comment:
We already have first module out, maybe it is good that we rephase it and say that there is no timeline about dynamic of publishing terraform modules. Something like this.

Reply:
Updated.